samps/contests/sumitMTC/config/sumit/problem.yaml

```diff
@@ -14,6 +14,15 @@ rights_owner:
 
 limits: 
    timeout: 30
+   memory: 2099
+
+sandbox-type: NONE
+# sandbox-type: PC2_INTERNAL_SANDBOX
+# sandbox-type: EXTERNAL_SANDBOX
+
+sandbox-command-line: '{:sandboxprogramname} {:memlimit} {:timelimit}'
+sandbox-program-name: 'samps/linux_sandbox'
```

Comment:
Am I correct that the above two lines will actually have no effect because of the fact that `sandbox-type: NONE` is selected a few lines above?  (If so, that's fine).

Reply:
If sandbox-type is NONE, then yeah, it will not use them, however, it still **does** populate the fields in the problem.  In addition, we actually do not support EXTERNAL types yet (there is an error message to that effect if you were to try and use an external one.  The GUI does not allow you to select it; the yaml on the other hand does, but it will later be reported as a problem).

` throw new Exception ("Unsupported sandbox type '" + sbType + "' in Problem configuration; cannot execute submission");`